WordPress/gutenberg
# [RNMobile] Display custom color value in mobile Cover Block color picker

## What?
Displays the hex color value when selecting a custom color on the mobile editor. Fixes:

* https://github.com/WordPress/gutenberg/issues/51283

Gutenberg Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5852

## Why?
For color contrast accessibility reasons, it is helpful to have the value of the color displayed in a text or number format other than the color itself. Also, displaying the hex value matches other Gutenberg color picker implementations on web and mobile (like the paragraph block, for example). 

## How?
When a custom color is selected in the Cover block, replace the "Select a color" label with the hex color value within the bottom sheet.

## Testing Instructions
1. Create a post on mobile
2. Create a Cover block
3. Move the selector to select a custom color
4. Note that the BottomSheet label changes from "Select a color" to the hex color value

## Screenshots or screencast 
(Note: the video player controls may obscure the actual PR changes at bottom of this video. Ensure you are not hovering over the video in order to see the hex value.)

Before | After
-|-
<video src="https://github.com/WordPress/gutenberg/assets/643285/c79150bd-2d3a-422c-bb83-c14a8f28547e" /> | <video src="https://github.com/WordPress/gutenberg/assets/643285/d76e840a-d0a3-4691-9912-fb30bb63cfc3" />
